typed-racket-doc/typed-racket/scribblings/reference/types.scrbl

```diff
@@ -354,6 +354,12 @@ corresponding to @racket[trest], where @racket[bound]
 
 @ex[(box "hello world")]
 
+@defform[(Read-Only-Boxof t)]{A read-only @rtech{box} of @racket[t].}
+
```

Comment:
Ok, I did not know that. I learned something. Still, doesn't it seem a little too confusing to refer to `const`? Reading through some stuff, I got confused about pointer to const versus const pointer, then eventually figured out what you meant. It'll be simpler to just show an example of a mutable box being passed as a read-only one.